tests/tracking/test_rest_tracking.py

```diff
@@ -803,6 +803,11 @@ def test_get_experiment(mlflow_client):
 
 
 def test_search_experiments(mlflow_client):
+    # To ensure the default experiment and non-default experiments have different creation_time
+    # for deterministic search results, send a request to the server and initialize the tracking
+    # store.
+    assert mlflow_client.search_experiments()[0].name == "Default"
```

Comment:
We have the following loop after this line. If we skip commented out this line, the first `craete_experiment` call would create two experiments (default and non-default). As experiment creation is pretty fast (takes a few ms on my machine), these two experiments may have the same cretion_time. This is more likely to happed on Windows because the time resoution is about 15 ms:

```
    for name, tags in experiments:
        time.sleep(0.001)
        experiment_ids.append(mlflow_client.create_experiment(name, tags=tags))
```